homeassistant/components/tesla/__init__.py

```diff
@@ -55,37 +60,72 @@
 def setup(hass, base_config):
```

Comment:
Again, I wanted to get the config flow PR in before working on the async. I thought the guidelines was to break up PRs.

Reply:
It doesn't work to break it up like that, since there will be errors. It's ok to refactor first to more easily use the home assistant async api in one PR and then add the config flow in a later PR. But the other way around doesn't work.